src/sparkx/Oscar.py

```diff
@@ -594,7 +594,7 @@ def uncharged_particles(self):
 
         for i in range(0, self.num_events_):
             self.particle_list_[i] = [elem for elem in self.particle_list_[i]
-                                        if (elem.charge == 0 and elem.charge != None)]
+                                        if (elem.charge == 0 and elem.charge != np.nan)]
```

Comment:
I just saw that there is also a `and elem.is_strange() != np.nan` needed in line 616 in the `strange_particles` filter.